R/nested_to_tree.R

```diff
@@ -17,11 +17,14 @@ create_tree <- function(text) {
 #' @param pd_nested A nested tibble.
 #' @return An object of class "Node" and "R6".
 #' @examples
+#' if (getRversion() >= 3.2) {
 #' code <- "a <- function(x) { if(x > 1) { 1+1 } else {x} }"
 #' nested_pd <- styler:::compute_parse_data_nested(code)
 #' initialized <- styler:::pre_visit(nested_pd, c(styler:::initialize_attributes))
 #' styler:::create_node_from_nested_root(initialized)
+#' }
 create_node_from_nested_root <- function(pd_nested) {
+  if (getRversion() < 3.2) stop_insufficient_r_version()
```

Comment:
Can we just check if the data.tree package is available here? They might start supporting R 3.1, or it still may be missing because it's Suggests only.